registrations/tests/test_registration_get.py

```diff
@@ -668,6 +670,42 @@ def test_registration_id_is_audit_logged_on_get_detail(user_api_client, registra
     ]
 
 
+@pytest.mark.django_db
+def test_list_should_order_by_id_by_default(user_api_client):
+    registration1 = RegistrationFactory()
+    registration2 = RegistrationFactory()
+    registration3 = RegistrationFactory()
+    expected_order = [registration1, registration2, registration3]
+
+    response = get_list(user_api_client)
+    assert response.status_code == status.HTTP_200_OK, response.json()
+
+    data = response.data["data"]
+    assert [item["id"] for item in data] == [item.pk for item in expected_order]
+
+
+@freezegun.freeze_time("2024-01-01")
+@pytest.mark.django_db
+def test_list_should_order_by_event_start_time(user_api_client):
+    registration_jun = RegistrationFactory(
+        event=EventFactory(start_time=datetime(2024, 6, 1))
```

Comment:
These create [warnings about naive datetimes](https://github.com/City-of-Helsinki/linkedevents/actions/runs/12431862884/job/34710102404#step:14:4077).